src/components/Explorer/DropdownMenu.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div :class="$style.dropdown">
+    <button :class="$style.button" @click="open">
+      <div :class="$style.dropdown_trigger">
+        <p :class="$style.title">{{ title }}</p>
+        <icon name="chevron-down" :class="$style.icon" />
+      </div>
+    </button>
+    <div v-if="isOpen" :class="$style.dropdown_menu">
+      <p
+        v-for="(content, index) in contents"
+        :key="index"
+        :class="$style.contents"
+      >
+        {{ content.str }}
+      </p>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { reactive } from 'vue'
+import Icon from '/@/components/UI/Icon.vue'
+
+export default {
+  name: 'DropdownMenu',
+  components: {
+    Icon
+  },
+  props: {
+    title: {
+      type: String,
+      required: true
+    },
+    contents: {
+      type: Array,
+      required: true
+    },
+    isOpen: {
+      type: Boolean,
+      required: true
+    }
+  },
+  emits: ['open'],
+  setup(props, { emit }) {
+    const state = reactive({ isOpen: false })
+
+    const open = (e: any) => {
+      emit('open', e)
+    }
```

Comment:
```suggestion
    const open = () => {
      emit('open')
    }
```
`any`型を使うとTSの意味がなくなるので極力使わないでください
`onclick`がcallbackで引数でとれる型は`MouseEvent`ですがそもそも呼び出してるコンポーネントで使っていないのでeventを渡さないでいいです
emitは第二引数以降を呼び出し元にあるcallback関数に引数として渡します